build.sh

```diff
@@ -1 +1,3 @@
-docker build -t kartoza/geogit:1.0 .
+#bin/bash
+git clone http://github.com/boundlessgeo/GeoGig.git
```

Comment:
We should have an alternative logic path in the docker file that checks out in the build process in the build context if the geogig dir is not present. Above build wont workon the docker build service.